src/blockstore/index.d.ts

```diff
@@ -0,0 +1,11 @@
+import { Block } from '@ipld/car/api'
+
+export interface Blockstore {
```

Comment:
Do we not have something like this already? Feels very familiar!

Reply:
We have a really more complex one: https://github.com/ipfs/js-ipfs-interfaces/blob/master/packages/interface-blockstore/src/types.d.ts